<launch>
  <arg name="debug" default="false"/>
  <arg name="rviz" default="false"/>
  <arg name="controller" default="true" />
  <arg name="gui" default="true"/>
  <arg name="run_mode" default="ethercattest"/>
  <arg name="urdfDampingmodel" default="false"/>
  
  <group if="$(arg controller)">
    <node name="tocabi_controller" pkg="tocabi_controller" type="tocabi_controller" output="screen">
      <param name="run_mode" value="$(arg run_mode)"/>
      <param name="debug" value="$(arg debug)" />
      <param name="rviz_pub" value="$(arg rviz)"/>
      <param name="ctime" value="1000"/>
      <param name="ifname" value="enp0s31f6"/>
      <param name="urdfAnkleRollDamping" value="$(arg urdfDampingmodel)"/>
    </node>
  </group>
</launch>
